libs/scripts/scripts.js

```diff
@@ -12,7 +12,7 @@
 
 import { getMetadata, decorateArea, loadBlock, loadLazy, loadStyle } from '../utils/utils.js';
 
-const LCP_BLOCKS = ['hero', 'home', 'marquee', 'section-metadata'];
+const LCP_BLOCKS = ['hero', 'home', 'marquee', 'media', 'section-metadata', 'z-pattern'];
```

Comment:
None of these additions should exist. If they're not true candidates for LCP, they should not be in here. I have yet to see where a z-pattern or a media block would be the LCP.